indico/modules/events/editing/client/js/components/EditingView/EditingFooter.jsx

```diff
@@ -0,0 +1,46 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+import logoURL from 'indico-url:assets.image';
+import contactURL from 'indico-url:core.contact';
+import tosURL from 'indico-url:legal.display_tos';
+
+import React from 'react';
+import {Image} from 'semantic-ui-react';
+import {Translate} from 'indico/react/i18n';
+
+import './EditingFooter.module.scss';
+
+export default function Editingfooter() {
+  return (
+    <div styleName="editing-footer">
+      <div styleName="logo">
+        <Image src={logoURL({filename: 'indico_small.png'})} styleName="image" />
+        <span styleName="text">
+          <Translate>Powered by </Translate>
+          <a href="http://getindico.io">Indico</a>
+        </span>
+      </div>
+      <div styleName="information">
+        <span>
+          <a href="https://learn.getindico.io">
```

Comment:
I think we have a configurable URL for this (since e.g. in case of CERN indico we want to point somewhere else)

Reply:
We have a URL for this but is the a dedicated endpoint? Apart from the one we have created in RB which returns other information as well.

Reply:
We should move it out from the RB one to a more generic place. Feel free to use the RB endpoint for the time being.

Reply:
We should actually separate the RB one into two, as it also contains the room sprite and other rb related stuff.